dbt/include/spark/macros/adapters.sql

```diff
@@ -229,9 +229,38 @@
   {% endfor %}
 {% endmacro %}
 
+{% macro get_column_comment_sql(column_name, column_dict) -%}
+  {% if (column_name|upper in column_dict) -%}
```

Comment:
Let's break this out into its own macro, something like `matched_column(column_name, column_dict)`.